c2corg_ui/views/xreport.py

```diff
@@ -20,6 +22,29 @@ def redirect_to_full_url(self):
 
     @view_config(route_name='xreports_view')
     def detail(self):
```

Comment:
I think that on profiles page, there is Activity feed with cached documents which are requested and rendered many times after scrolling down. I am querying one document once.